src/ReflectionProperty.php

```diff
@@ -106,33 +107,18 @@ public function __debugInfo(): array
         ];
     }
 
-    /**
-     * Return string representation of this little old property.
-     *
-     * @return string
-     */
-    public function __toString()
```

Comment:
Why you have decided to remove this method? Now it requires loading of internal metadata of reflection, which is against idea of this library to provide information without loading core data ((

Reply:
Is the `ValueResolver\NodeExpressionResolver` ideal for the `__toString()` method for `ReflectionProperty`? I tried to re-add it, but it looks very different in PHP 8.0 and 8.1

Reply:
Nevermind, I think I have to apply some custom logic

Reply:
To be honest, this method is not used too much, so if you have troubles with it - then just simple skip it, it doesn't deserve too much time

Reply:
Re-added, but not fully. It's not that important
https://github.com/goaop/parser-reflection/pull/122/commits/f8f982055a80da6707aab564167d9b024b314c00